uk.co.compucorp.civicrm.hrcore/gulp/tasks/backstopjs.js

```diff
@@ -139,7 +140,10 @@ function runBackstopJS (command) {
               configPath: BACKSTOP_DIR_PATH + destFile,
               filter: argv.filter
             }).catch(_.noop).then(function () { // equivalent to .finally()
-              gulp.src(BACKSTOP_DIR_PATH + destFile, { read: false }).pipe(clean());
+              gulp
+                .src(BACKSTOP_DIR_PATH + destFile, { read: false })
+                .pipe(clean())
+                .pipe(notify({ message: 'Backstop has finished', sound: true }));
```

Comment:
Most importantly you need to show a notification when something goes wrong (see if there is some kind of status for the notifications to mark if something has failed/succeeded)

Reply:
Done